src/components/dialogs/dialogAlert/dialogAlertContent/dialogAlertContent.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { DialogAlertRoot } from '../dialogAlertRoot';
+import { DialogAlertTrigger } from '../dialogAlertTrigger';
+import { DialogAlertContent } from './dialogAlertContent';
+
+const meta: Meta<typeof DialogAlertContent> = {
+    title: 'components/Dialogs/DialogAlertContent',
+    component: DialogAlertContent,
+    tags: ['autodocs'],
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/P0GeJKqILL7UXvaqu5Jj7V/v1.1.0?type=design&node-id=13558-17025&mode=design&t=2c10sWNHo18bHNd3-4',
+        },
+    },
+};
+
+type Story = StoryObj<typeof DialogAlertContent>;
+
+/**
+ * Default usage of the `DialogAlertContent` component
+ */
+export const Default: Story = {
+    args: {
+        title: 'Alert Title',
+        actionLabel: 'Action',
+        cancelLabel: 'Cancel',
+    },
+    render: (props) => (
+        <DialogAlertRoot>
+            <DialogAlertTrigger>Trigger alert</DialogAlertTrigger>
+            <DialogAlertContent {...props}>
+                <p>This is an alert!</p>
+            </DialogAlertContent>
+        </DialogAlertRoot>
```

Comment:
I think we can improve the components composition by dropping the uncontrolled case (which we never use right now and won't use later). In my opinion currently too much logic is being handled by the `content` component and it could be split instead to `title`, `content` & `footer`. By dropping the uncontrolled case, we could have something like:
```
<DialogAlert.Root>
  <DialogAlert.Title />
  <DialogAlert.Content />
  <DialogAlert.Footer />
</DialogAlert.Root>
```
Let's maybe sync later today on this :)